server/src/main/java/org/spine3/server/entity/storage/EntityRecordWithColumns.java

```diff
@@ -35,6 +38,17 @@
  */
 public final class EntityRecordWithColumns {
 
+    private static final Maps.EntryTransformer<String, Column.MemoizedValue<?>, Column<?>>
+    columnTransformer = new Maps.EntryTransformer<String, Column.MemoizedValue<?>, Column<?>>() {
```

Comment:
Can we extract this code into a static method instead?

Maybe having an internal class would be even better. The aim is 

a) to get rid of ` Maps.EntryTransformer<String, Column.MemoizedValue<?>, Column<?>>` definition, which is really long;
b) to avoid long field initialization, that is hard to read.